cocos2d/CCTMXXMLParser.m

```diff
@@ -486,35 +482,23 @@ - (void)parser:(NSXMLParser *)parser didEndElement:(NSString *)elementName names
 		_storingCharacters = NO;
 
 		CCTiledMapLayerInfo *layer = [_layers lastObject];
-
-		unsigned char *buffer;
-		len = base64Decode((unsigned char*)[_currentString UTF8String], (unsigned int) [_currentString length], &buffer);
-		if( ! buffer ) {
-			CCLOG(@"cocos2d: TiledMap: decode data error");
-			return;
-		}
+        
+        NSData *decoded = [[NSData alloc] initWithBase64Encoding:_currentString];
+        NSAssert(decoded, @"Could not decode tilemap data.");
 
 		if( _layerAttribs & (TMXLayerAttribGzip | TMXLayerAttribZlib) ) {
```

Comment:
Looks like more indentation oddities around here. Are they worth pointing out in code reviews?